docs/mindsdb-docs/docs/getting-started.md

```diff
@@ -1,106 +1,96 @@
 # Getting Started
 
-MindsDB integrates with the most popular databases and also with the DBT and MLflow workflow you already have.
+MindsDB can be integrated with the most popular databases, as well as with the DBT and MLFlow workflows.
 
-<sup><sub>To try MindsDB right away, without bringing your own data or model, check out our [Quick Start Guide](/).</sub></sup>
+To try out MindsDB right away without bringing in your own data or models, follow our [Quickstart Guide](/).
 
 1. Choose your MindsDB installation path.
 
     === "MindsDB Cloud"
 
-        !!! example ""
-            Create your [free MindsDB Cloud account](https://cloud.mindsdb.com/signup).
-
-            ![Cloud Signup](/assets/cloud-signup.png)
+        Create your [free MindsDB Cloud account](https://cloud.mindsdb.com/signup).
 
     === "Docker"
 
-        !!! example ""
-            To get started with a Docker installation, begin with our [Docker instructions](/deployment/docker).
+        To get started with a Docker installation, follow the MindsDB installation instructions using [Docker](/deployment/docker).
+
+    === "pip"
+
+        You can also install MindsDB from source using [pip](setup/self-hosted/pip/source/).
 
 1. Open your SQL client and connect to MindsDB.
 
-    <sup><sub>If you do not already have a preferred SQL client, we recommend [DBeaver Community Edition](https://dbeaver.io/download/)</sub></sup>
+    !!! Tip ""
+        If you do not have a preferred SQL client yet, we recommend [DBeaver Community Edition](https://dbeaver.io/download/). Follow [this guide](/connect/dbeaver/) on how to connect to MindsDB from DBeaver.
```

Comment:
This was before we got our SQL Editor, we can say we prefer MindsDBs SQL Editor or Dbeaver

Reply:
Here is the corrected tip (in both Quickstart and Getting Started sections).
![image](https://user-images.githubusercontent.com/109554435/186141498-ad0f89b2-c208-4f0e-8dd8-23e67e3b8022.png)